### PR TITLE
feat(oauth): support for Slack

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: core/src/oauth/connection.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 use crate::oauth::providers::{
     confluence::ConfluenceConnectionProvider, google_drive::GoogleDriveConnectionProvider,
```
Comment (spolu) [inline core/src/oauth/connection.rs:2]: meed to merge with the one below :)

**Review by spolu — APPROVED** [review]
Looking sharp

File: core/src/oauth/connection.rs (hunk 1 above)
Comment (fontanierh) [reply core/src/oauth/connection.rs:2]: WDYM ?

Comment (spolu) [reply core/src/oauth/connection.rs:2]: Imports should be grouped

Comment (fontanierh) [reply core/src/oauth/connection.rs:2]: In what way are they not groupped right now?

Comment (fontanierh) [reply core/src/oauth/connection.rs:2]: Nvm me i didnt see the github one below

**Comment by fontanierh** [comment]
Properly tested, good to go

**Pull request merged**